Sources/TableViewDriver.swift

```diff
@@ -255,7 +255,7 @@ extension TableViewDriver: UITableViewDelegate {
 
     /// :nodoc:
     public func tableView(_ tableView: UITableView, heightForRowAt indexPath: IndexPath) -> CGFloat {
-        return self.tableViewModel?[indexPath]?.rowHeight ?? 44
+        return self.tableViewModel?[indexPath]?.rowHeight ?? self.tableViewModel?.defaultRowHeight ?? 44
```

Comment:
By the time this method is called, we should have a non-nil `tableViewModel` (*I think*)

can we adjust to:

```swift
guard let tableViewModel = self.tableViewModel else { return 0 }
return tableViewModel[indexPath]?.rowHeight ?? tableViewModel.defaultRowHeight
```

(eventually, i think we'll make `self.tableViewModel` non-optional)